website/docs/r/shared_image.html.markdown

```diff
@@ -90,6 +92,16 @@ A `identifier` block supports the following:
 
 * `sku` - (Required) The Name of the SKU for this Gallery Image.
 
+---
+
+A `purchase_plan` block supports the following:
+
+* `name` - (Required) The Purchase Plan Name for this Shared Image.
```

Comment:
We'll want to add that `Changing this forces a new resource to be created.` for each of these attributes

Reply:
Done